ui/page/start_page.go

```diff
@@ -259,7 +260,12 @@ func (sp *startPage) HandleUserInteractions(gtx C) {
 	}
 
 	if sp.addWalletButton.Clicked(gtx) {
-		createWalletPage := components.NewCreateWallet(sp.Load, func() {
+		isFirstWallet := sp.AssetsManager.LoadedWalletsCount() == 0
+		createWalletPage := components.NewCreateWallet(sp.Load, func(newWallet sharedW.Asset) {
+			if newWallet != nil && isFirstWallet {
```

Comment:
why does it matter if it's the first wallet, all wallets should have auto sync enabled